content/to-other-languages/typescript-for-c-sharp-developers.md

```diff
@@ -4,3 +4,152 @@ showMetadata: false
 editable: true
```

Comment:
I think you miss a title of this document.

Reply:
the title is already written by someone?